playwright/.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "parserOptions": {
+    "project": "./tsconfig.json"
+  },
+  "rules": {
+    "@typescript-eslint/no-floating-promises": "error"
+  }
```

Comment:
**question**: Interesting, we have it turned off for the dashboard coffee. why do you add it for playwright?

Reply:
Suggestion from friendly team ;)  - help them to spot missing await's in PW tests code